image.yaml

```diff
@@ -4,41 +4,33 @@ schema_version: 1
 # to give a meaningful error on an attempt to install Spark in
 # a complete image via S2I
 
-version: 2.4-latest
+version: 2.4.6 
 release: community
 name: radanalyticsio/openshift-spark
-from: centos:centos7
+from: centos:8
 labels:
     - name: maintainer
-      value: Chad Roberts <croberts@redhat.com>
+      value: Trevor McKay <tmckay@redhat.com>
     - name: sparkversion
-      value: 2.4.6
+      value: 3.0.0
     - name: "io.openshift.s2i.scripts-url"
       value: "image:///usr/libexec/s2i"
-envs:
-    - name: PATH
-      value: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/spark/bin
-    - name: SPARK_HOME
-      value: /opt/spark
-    - name: SPARK_INSTALL
-      value: /opt/spark-distro
-    - name: STI_SCRIPTS_PATH
-      value: /usr/libexec/s2i
-    - name: SCL_ENABLE_CMD
-      value: ""
+
+packages:
+    install:
+        - java-1.8.0-openjdk
```

Comment:
should check on this for spark 3

Reply:
Definitely we should move to java 11 for two reasons:

- Java 8 is EOL by Oracle for a while, although Red Hat will support until 2026
- Java 11 has better support in containerized environments